public/app/features/admin/AdminSettings.tsx

```diff
@@ -74,4 +72,4 @@ const mapStateToProps = (state: StoreState) => ({
   navModel: getNavModel(state.navIndex, 'server-settings'),
 });
 
-export default hot(module)(connect(mapStateToProps)(AdminSettings));
```

Comment:
Will keeping `hot(module)` break anything? We use those in Enterprise so wonder if they'd be also removed there.